album/undertale-deltarune-fan-encore-cd.yaml

```diff
@@ -0,0 +1,165 @@
+Album: UNDERTALE + deltarune FAN ENCORE CD
+Directory: undertale-deltarune-fan-encore-cd
+Directory Suffix: fan-encore
+Suffix Track Directories: true
+Artists:
+- Toby Fox
+Date: May 21, 2019
+#Date Added: February 5, 2024
+#URLs:
+#- 
+Color: '#ffffff'
+Cover Artists:
+- maitou
+Cover Art File Extension: png
+Groups:
+- Toby Fox
+- UNDERTALE & DELTARUNE
+Art Tags:
+- Frisk
+- Kris (Dark World)
+---
+Section: DELTARUNE Trial Tracks
+---
+Track: DOGTARUNE (Trailer)
+Duration: 0:16
+#URLs:
+#- 
+Commentary: |-
+    <i>Nintendo DREAM:</i> (Booklet)
+
+    ("DELTARUNE Chapter 1" Launch Trailer)
+
+    <i>Toby Fox:</i> (UNDERTALE + deltarune FAN ENCORE CD - Booklet)
+
+    この曲を作ったときは、なんも考えてなかったです。
+---
+Track: Beginning
+Always Reference By Directory: true
+Duration: 0:59
+Main Release: track:beginning-deltarune
+Commentary: |-
+    <i>Nintendo DREAM:</i> (Booklet)
+
+    (From "DELTARUNE Chapter 1")
+
+    <i>Toby Fox:</i> (UNDERTALE + deltarune FAN ENCORE CD - Booklet)
+
+    手を振っている ご近所さん<br>
```

Comment:
we need to get someone to translate the japanese commentary on the wiki at some point....